sdk/typescript/lib/subapis/status.ts

```diff
@@ -0,0 +1,13 @@
+import { StatusApi } from "../generated";
+
+export class Status {
+    constructor(public innerClient: StatusApi) {}
+
+    getGatewayStatus() {
```

Comment:
Do you think it's worth explicitly annotating types on method returns so that it's clearer if people click through? :)

Reply:
Hm... Yes I think it is